docs/getting_started.md

```diff
@@ -1,17 +1,18 @@
 # Getting started
 
 ## Get VaSeBuilder
-VaSeBuilder can be obtained by either cloning the repository via ```git clone```, downloading a zipped archive, or by downloading the latest release. In the future we aim to make VaSeBuilder available as a pip package as well.
+VaSeBuilder can be obtained by either cloning the [repository](https://github.com/molgenis/VaSeBuilder) via ```git clone```, downloading a zipped archive, or by downloading the [latest release](https://github.com/molgenis/VaSeBuilder/releases). In the future we aim to make VaSeBuilder available as a pip package as well.
```

Comment:
I would only "promote/advertise" the releases. The geeks will already know how to fork or clone and use the latest unreleased code; no need to explain that here.